backend/dal/Helpers/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -57,6 +58,7 @@ public static IServiceCollection AddPimsDalRepositories(this IServiceCollection
         /// <returns></returns>
         public static IServiceCollection AddPimsDalServices(this IServiceCollection services)
         {
+            // Todo: move services to the API project
```

Comment:
tech debt for this? we don't have a great track record on following up on these if all we have is a todo.

Reply:
Once the warnings are cleared (pending PR) we should enable the TODO  warning and create tickets for each of those if they dont exist